helpers/matrix.yml

```diff
@@ -11,28 +11,34 @@ ES_SUITE:
   - multi
   - oss
   - security
+  - upgrade
 KIBANA_SUITE:
   - default
   - oss
   - security
+  - upgrade
 FILEBEAT_SUITE:
   - default
   - oss
   - security
+  - upgrade
 METRICBEAT_SUITE:
   - default
   - oss
   - security
+  - upgrade
```

Comment:
I see the `FROM` is commented out in the metricbeat upgrade makefile, is it ready to be added without the 7.10.0 helm-chart release being out yet?

Reply:
Actually the `test` target is also commented in https://github.com/elastic/helm-charts/pull/907/files?file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#diff-24429fd964f0f618b6c9c57dd7ea1977aa62316fbce76bef1025b24a1da762edR14

So the Metricbeat upgrade will pass without doing anything for now. until we release 7.10.0 then uncomment `FROM` and `test` target.

Reply:
Ah I see, I think it would be good to remove it from this file as well to avoid spinning up a worker/job that doesn't actually do anything and gives the appearance of passing.

Reply:
done in e7a2df5